tensorflow_encrypted/protocol/pond.py

```diff
@@ -262,11 +273,16 @@ def helper(v: tf.Tensor):
 
     def define_output(
         self,
+        player: Union[str, Player],
         xs: Union['PondPrivateTensor', List['PondPrivateTensor']],
-        receiver: OutputReceiver,
+        post_fn: Callable[..., Any],
```

Comment:
this guy is a `Callable[tf.Tensor, ...]`if i understand that syntax correct

Reply:
Not really, it can take any number of parameters:

`def post_fn(*args: tf.Tensor) -> Any`

But I didn't find any way to type this `*args` in `Callable` definition. I would need something like `Callable[...:tf.Tensor, Any]` or `Callable[StarArg: tf.Tensor, Any]` but it's not working

Also, I just noticed I didn't rename it, what name would you prefer? `outputter_fn`?

Reply:
yeah, `outputter_fn` works 👍